### PR TITLE
Vijay edit

### PR DESCRIPTION
**Pull request by Vijay1126**
What Avinash had done:
image

The Figma Design:
image

What I have done:
image

### CODE REVIEW

File: components/DashboardChart.js [hunk 1]
```diff
@@ -57,67 +79,211 @@ export default class FoodTrends extends React.PureComponent {
       };
 
     render() {
+    const statusArray = [.5,1,1,1];
+    var currentProgress = statusArray.reduce(function(a, b) { return a + b; }, 0)/4;
+    const data = Array.apply(null, Array(40)).map(Number.prototype.valueOf, 1);
+    
+
+    // Replace with your fill method...
+    const getFill = (index) => {
```
Comment (nandukalidindi) [inline components/DashboardChart.js:88]: This method can be updated to be a little more generic like we discussed on the call.

File: components/DashboardChart.js [hunk 2]
```diff
@@ -57,67 +79,211 @@ export default class FoodTrends extends React.PureComponent {
...
-                <TouchableOpacity style={styles.fab2}>
-                    <MaterialCommunityIcons name="food" size={16} color="white" />
-                </TouchableOpacity>
-                <TouchableOpacity style={styles.fab3}>
-                    <MaterialCommunityIcons name="power-sleep" size={16} color="white" />
-                </TouchableOpacity>
-                <TouchableOpacity style={styles.fab4}>
-                    <FontAwesome5 name="running" size={14} color="white" />
-                </TouchableOpacity>
+                <ForeignObject
```
Comment (nandukalidindi) [inline components/DashboardChart.js:157]: May be everything inside the Pie chart can be moved into a separate component which takes in all the necessary parameters as props.

File: components/DashboardChart1.js [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    },
+    fab2: {
+        position: 'absolute',
+        margin: 16,
+        right: 160,
+        bottom: 270,
+        borderRadius:1,
+        backgroundColor: 'transparent'
+      },
+      fab3: {
```
Comment (nandukalidindi) [inline components/DashboardChart1.js:32]: Please avoid using indexed class names, might get confusing eventually.

**Comment by nandukalidindi** [comment]
Please try to separate out your changes as much as you can. Don't push all your changes into one PR. I added a READY_TO_MERGE label. Please add that if your PR is ready to be merged.

**Comment by nandukalidindi** [comment]
Merging this PR now, please issue a separate PR for upcoming changes and keep them simple.

**Comment by Vijay1126** [comment]
Will do, thank you @nandukalidindi

**Pull request merged**